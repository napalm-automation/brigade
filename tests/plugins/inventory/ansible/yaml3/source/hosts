---
all:
  vars:
    ansible_python_interpreter: python
    ansible_connection: network_cli
    ansible_network_os: ios
    ansible_ssh_common_args: '-o ProxyCommand="ssh -W %h:%p -p 10000 guest@10.105.152.50"'  # install ssh public key on proxy host !!!
    env: staging
  children:
    virl:
      children:
        asa:
          vars:
            cisco_asa: True
            network_os: asa
            ansible_network_os: asa
            ansible_become: yes
            ansible_become_method: enable
            ansible_persistent_command_timeout: 30
          hosts:
            asa1:
              ansible_host: 10.255.0.2
              dot1x: True
        ios:
          vars:
            network_os: ios
            nornir_nos: ios
            platform: ios
          children:
            routers:
              hosts:
                iosv-1:
                  ansible_host: 10.255.0.12
                iosv-2:
                  ansible_host: 10.255.0.13
            access:
              hosts:
                access1:
                  ansible_host: 10.255.0.6
            dist:
              hosts:
                dist1:
                  ansible_host: 10.255.0.5
                dist2:
                  ansible_host: 10.255.0.11
