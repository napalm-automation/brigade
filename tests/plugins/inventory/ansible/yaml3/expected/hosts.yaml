access1:
  connection_options: {}
  data: {}
  groups:
  - access
  hostname: 10.255.0.6
  password: null
  platform: null
  port: null
  username: null
asa1:
  connection_options: {}
  data:
    dot1x: true
  groups:
  - asa
  hostname: 10.255.0.2
  password: null
  platform: null
  port: null
  username: null
dist1:
  connection_options: {}
  data: {}
  groups:
  - dist
  hostname: 10.255.0.5
  password: null
  platform: null
  port: null
  username: null
dist2:
  connection_options: {}
  data: {}
  groups:
  - dist
  hostname: 10.255.0.11
  password: null
  platform: null
  port: null
  username: null
iosv-1:
  connection_options: {}
  data: {}
  groups:
  - routers
  hostname: 10.255.0.12
  password: null
  platform: null
  port: null
  username: null
iosv-2:
  connection_options: {}
  data: {}
  groups:
  - routers
  hostname: 10.255.0.13
  password: null
  platform: null
  port: null
  username: null
